src/lib.rs

```diff
@@ -366,3 +366,50 @@ impl<T: View> ViewWrapper for AlignedView<T> {
         self.view.important_area(self.last_size) + self.offset
     }
 }
+
+#[cursive_core::blueprint(AlignedView::new(view, alignment))]
+struct Blueprint {
+    view: cursive_core::views::BoxedView,
+    alignment: Align,
+}
+
+cursive_core::manual_blueprint!(with align, |config, context| {
+    let alignment = context.resolve(config)?;
+    Ok(move |view| AlignedView::new(view, alignment))
+});
+
+cursive_core::manual_blueprint!(with align_top_left, |_config, _context| {
+    Ok(|view| AlignedView::with_top_left(view))
+});
+
+cursive_core::manual_blueprint!(with align_top_center, |_config, _context| {
+    Ok(|view| AlignedView::with_top_center(view))
+});
```

Comment:
This allows using:

```json
{
    "Panel": {
        "view": "DummyView",
        "with": [
            "align_top_center"
        ]
    }
}

Reply:
And since `Panel` also implements its own with-blueprint, it could as well be:

```json
{
    "DummyView": {
        "with": [
            "panel",
            "align_top_center"
        ]
    }
}
```